pom.xml

```diff
@@ -22,15 +22,18 @@
         <!--Semantic version -->
         <apiguardian-api.version>1.1.0</apiguardian-api.version>
 
-        <!--Dependencies -->
-        <cucumber-expressions.version>10.2.2</cucumber-expressions.version>
-        <datatable.version>3.3.1</datatable.version>
-        <create-meta.version>1.2.0</create-meta.version>
-        <tag-expressions.version>3.0.0</tag-expressions.version>
-        <!-- When ever messages is updated run `make update-compatibility` -->
-        <messages.version>12.4.0</messages.version>
-        <gherkin.version>15.0.0</gherkin.version>
-        <html-formatter.version>7.2.0</html-formatter.version>
+        <!-- io.cucumber dependencies - use semver version ranges for these -->
+        <cucumber-expressions.version>[10.2.2,11.0.0)</cucumber-expressions.version>
+        <datatable.version>[3.4.0,4.0.0)</datatable.version>
+        <create-meta.version>[2.0.0,3.0.0)</create-meta.version>
+        <tag-expressions.version>[3.0.0,4.0.0)</tag-expressions.version>
+        <!--
+        Whenever messages is updated run `MSG_VERSION=... make update-compatibility`,
+        specifying the latest released version number for messages. 
+        -->
+        <messages.version>[13.0.1,14.0.0)</messages.version>
+        <gherkin.version>[15.0.0,16.0.0)</gherkin.version>
```

Comment:
This won't work. We shade Gherkin in to avoid conflicts with tooling that uses older versions of Gherkin.